examples/_toc.yml

```diff
@@ -4,6 +4,18 @@
 format: jb-book
 root: Getting_started
 parts:
+  - caption: pytket documentation
+    chapters:
+    - url: https://tket.quantinuum.com/api-docs/
+      title: pytket API docs
+    - url: pytket extensions <https://tket.quantinuum.com/api-docs/extensions.html> 
```

Comment:
Why are you writing a name in the link here as well?

Reply:
see above